geomesa-hbase/geomesa-bigtable-tools/bin/geomesa-bigtable

```diff
@@ -24,15 +24,7 @@ if [[ -n "$HADOOP_HOME" ]]; then
 fi
 
 # Warn user about missing environment variables
-if [[ -z "$HADOOP_HOME" ]]; then
-  message="Warning: you have not set HADOOP_HOME as an environment variable."
-  message="${message}\nGeoMesa tools will not run without the appropriate Hadoop jars in the tools classpath."
-  message="${message}\nPlease ensure that those jars are present in the classpath by running 'geomesa-bigtable classpath' ."
-  message="${message}\nTo take corrective action, please place the necessary jar files in the lib directory of geomesa-tools."
-  echo >&2 ""
-  echo -e >&2 "$message"
-  echo >&2 ""
-elif [[ -z "$HADOOP_CONF_DIR" ]]; then
+if [[ -z "$HADOOP_CONF_DIR" ]]; then
```

Comment:
Yeah, there are probably a few cases where care if it's empty vs set as we could give some more helpful messages. Who ever does this will have to be careful with the logic though, I find it can be very confusing. I think for the HADOOP_CONF_DIR it should be `[[ -z "${HADOOP_CONF_DIR:+x}" ]]`
e.g.
`$ HADOOP_CONF_DIR=`
`$ if [[ -z "${HADOOP_CONF_DIR:+x}" ]]; then echo "We need to Set HADOOP_CONF_DIR"; fi`
`We need to Set HADOOP_CONF_DIR`
`$ unset HADOOP_CONF_DIR`
`$ if [[ -z "${HADOOP_CONF_DIR:+x}" ]]; then echo "We need to Set HADOOP_CONF_DIR"; fi`
`We need to Set HADOOP_CONF_DIR`
`$ HADOOP_CONF_DIR=/abc/`
`$ if [[ -z "${HADOOP_CONF_DIR:+x}" ]]; then echo "We need to Set HADOOP_CONF_DIR"; fi`
`$`